package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "",
   "scripts": {
-    "lint": "cross-env-shell cd $INIT_CWD && tsqllint"
+    "lint": "cross-env-shell cd $GITHUB_WORKSPACE && ls && tsqllint"
```

Comment:
Does this mean it will only work if tsqllint is in the root of the repository?

Reply:
It shouldn't. Now that you bring it up though, I'm not 100% sure of that. I can try throwing this action into the database project and test using it locally. That'll give me full confidence.

Reply:
Ok, now I'm 100% confidant! The tsqllint command is still recognized. I did end up having a problem with the script syntax though, so it's good you asked the question and sent me down this road.